spring-cql/pom.xml

```diff
@@ -101,6 +101,12 @@
 			<artifactId>commons-lang3</artifactId>
 			<scope>test</scope>
 		</dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+            <version>1.1.7</version>
```

Comment:
Thanks for pointing out. That was also the case for Cassandra 2. We should rather exclude the dependency (see https://github.com/spring-projects/spring-data-cassandra/pull/51/files#diff-38c92dfe92517d397253b3b16a4ff02bL85) so we don't have to maintain logback versions.